kyuubi-events/src/main/scala/org/apache/kyuubi/events/EventBus.scala

```diff
@@ -68,6 +70,10 @@ object EventBus extends Logging {
   def registerAsync[T <: KyuubiEvent: ClassTag](et: EventHandler[T]): EventBus =
     defaultEventBus.registerAsync[T](et)
 
+  def deregisterAll(): Unit = {
+    defaultEventBus.deregisterAll()
```

Comment:
synchronized

Reply:
Which one are you suggesting? Synchronized on the object of EventBus instance or on the defaultEventBus instance?

```
def deregisterAll(): Unit = synchronized {
```
or
```
def deregisterAll(): Unit = defaultEventBus.synchronized {
```

Reply:
the former